packages/web-react/src/components/TextArea/README.md

```diff
@@ -0,0 +1,47 @@
+# TextArea
+
+TextArea enables the user to add longer text to a form. It has textarea, label,
+and an optional message. It could be disabled or have an error state. The label could be hidden
+and show if the textarea is required.
+
+```jsx
+<TextArea id="example" name="example" validationState="error" message="validation failed" isRequired />
+<TextArea id="example" name="example" validationState="error" message="validation failed" isRequired />
+<TextArea id="example" name="example" validationState="error" message="validation failed" isRequired />
```

Comment:
Is there a reason why there are three identical examples below each other?

Reply:
_Copypasta_ again. 🤦🏻‍♀️ Thanks!